website/docs/cloud-docs/integrations/service-now/service-catalog-terraform/developer-reference.mdx

```diff
@@ -49,20 +49,28 @@ Server Development > Script Include.
 | Class Name            | Description                                                |
 | --------------------- | ---------------------------------------------------------- |
 | `tf_config`           | Helper to pull values from the SN Terraform Configs Table  |
+| `tf_get_workspace`    | Client-callable script to retrieve workspace data          |
```

Comment:
I think it's important to leave "client-callable" here because it's a specific type of script in ServiceNow, which makes it different from other backend scripts.